<sdf version='1.6'>
 	 <world name='default'>
 	 	 <light name='sun' type='directional'>
 	 	 	 <cast_shadows>1</cast_shadows>
 	 	 	 <pose frame=''>0 0 10 0 -0 0</pose>
 	 	 	 <diffuse>0.8 0.8 0.8 1</diffuse>
 	 	 	 <specular>0.1 0.1 0.1 1</specular>
 	 	 	 <attenuation>
 	 	 	 	 <range>1000</range>
 	 	 	 	 <constant>0.9</constant>
 	 	 	 	 <linear>0.01</linear>
 	 	 	 	 <quadratic>0.001</quadratic>
 	 	 	 </attenuation>
 	 	 	 <direction>0.9 0.5 -1</direction>
 	 	 </light>

 	 	 <include>
 	 	 	 <uri>model://Sign</uri>
 	 	 	 <pose>12 1 0 0 0 0</pose>
 	 	 	 <static>1</static>
 	 	 </include>

 	 	 <include>
 	 	 	 <uri>model://Sign</uri>
 	 	 	 <pose>-4 1 0 0 0 0</pose>
 	 	 	 <static>1</static>
 	 	 </include>	 	 <include>
 	 	 	 <uri>model://heightmap</uri>
 	 	 </include>

     <scene>
  	 	 	 <background>160 210 255 1 </background>
        <ambient>100 100 100 1</ambient>
  	 	 </scene>

 	 	 <population name="sticks">
 	 	 	 <model name="Red_stick">
 	 	 	 	 <include>
 	 	 	 	 	 <static>0</static>
 	 	 	 	 	 <uri>model://Red_stick</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	<pose>4.100000 -0.350000 0.1 0 0 0</pose>
 	 	 	<distribution>
 	 	 	 	 <type>grid</type>
 	 	 	 	 <rows>3</rows>
 	 	 	 	 <cols>2</cols>
 	 	 	 	 <step>6.200000 0.700000 0</step>
 	 	 	 </distribution>
 	 	 </population>

	 	 <population name="Mound">
 	 	 	 <model name="Mound">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Mound</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	<pose>4.100000 -0.350000 -0.245 0 0 0</pose>
 	 	 	<distribution>
 	 	 	 	 <type>grid</type>
 	 	 	 	 <rows>3</rows>
 	 	 	 	 <cols>2</cols>
 	 	 	 	 <step>6.200000 0.700000 0</step>
 	 	 	 </distribution>
 	 	 </population>

	 	<population name="Mound_cyl">
 	 	 	 <model name="Mound_cyl">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Mound_cyl</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.100000 -0.350000 -0.2 0 0 0</pose>
 	 	 	<distribution>
 	 	 	 	 <type>grid</type>
 	 	 	 	 <rows>3</rows>
 	 	 	 	 <cols>24</cols>
 	 	 	 	 <step>0.250000 0.700000 0</step>
 	 	 	 </distribution>
 	 	 </population>

	 	<population name="Vegetable">
 	 	 	 <model name="Leek">
 	 	 	 	 <include>
 	 	 	 	 	 <static>0</static>
 	 	 	 	 	 <uri>model://Leek</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>4.100000 -0.350000 0.2 0 0 0</pose>
 	 	 	<distribution>
 	 	 	 	 <type>grid</type>
 	 	 	 	 <rows>3</rows>
 	 	 	 	 <cols>24</cols>
 	 	 	 	 <step>0.250000 0.700000 0</step>
 	 	 	 </distribution>
 	 	 </population>

	 	 <population name="Trees1">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>17 0 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>5 20 3</size>
 	 	 	 </box>
 	 	 	 <model_count>5</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>

 	 	 <population name="Trees2">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>0 -17 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>30 6 3</size>
 	 	 	 </box>
 	 	 	 <model_count>5</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>

 	 	 <population name="Trees3">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>0 15 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>30 6 3</size>
 	 	 	 </box>
 	 	 	 <model_count>7</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 	 </population>

 	 	 <population name="Trees4">
 	 	 	 <model name="Tree">
 	 	 	 	 <include>
 	 	 	 	 	 <static>1</static>
 	 	 	 	 	 <uri>model://Tree</uri>
 	 	 	 	 </include>
 	 	 	 </model>
 	 	 	 <pose>-15 0 -0.6 0 0 0</pose>
 	 	 	 <box>
 	 	 	 	 <size>10 30 3</size>
 	 	 	 </box>
 	 	 	 <model_count>8</model_count>
 	 	 	 <distribution>
 	 	 	 	 <type>random</type>
 	 	 	 </distribution>
 	 	 </population>

	 </world>
 </sdf>
